kubernetes/chart/zulip/templates/NOTES.txt

```diff
@@ -0,0 +1,15 @@
+1. Zulip should now be available on:
+{{- if .Values.ingress.enabled }}
+{{- range .Values.ingress.hosts }}
+  http://{{ .host }}
+{{- end }}
+{{- else }}
+  export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app={{ template "zulip.name" . }},release={{ .Release.Name }}" -o jsonpath="{.items[0].metadata.name}")
+  echo "Visit http://127.0.0.1:8080 to use your application"
+  kubectl port-forward $POD_NAME 8080:80
+{{- end }}
+
+2. To create a realm so you can sign in:
+
+  kubectl -n {{ .Release.Namespace }} exec -it $(kubectl get pods --namespace {{ .Release.Namespace }} -l "app.kubernetes.io/instance={{ .Release.Name }}-{{ template "zulip.name" . }}" -o jsonpath="{.items[0].metadata.name}") -c zulip -- sudo -u zulip /home/zulip/deployments/current/manage.py generate_realm_creation_link
```

Comment:
I can't make this work for me. I followed the README.md and all pods install fine, I can reach the login page.
But I can't create a realm: 

```
2. To create a realm so you can sign in:

  kubectl -n zulip exec -it $(kubectl get pods --namespace zulip -l "app.kubernetes.io/instance=zulip-zulip" -o jsonpath="{.items[0].metadata.name}") -c zulip -- sudo -u zulip /home/zulip/deployments/current/manage.py generate_realm_creation_link
```

The instance label filter needs to change from ` "app.kubernetes.io/instance=zulip-zulip"` to ` "app.kubernetes.io/instance=zulip"` otherwise it won't match any pod.
Even with the changed filter, `"{.items[0].metadata.name}"` will give me the memcached pod, not the main zulip one:

```
$ kubectl get pods --namespace zulip -l "app.kubernetes.io/instance=zulip" -o jsonpath="{.items[0].metadata.name}" 
zulip-memcached-84ffbdf7d6-xr4nl
```

which will lead to this:

```
$ kubectl -n zulip exec -it $(kubectl get pods --namespace zulip -l "app.kubernetes.io/instance=zulip" -o jsonpath="{.items[0].metadata.name}") -c zulip -- sudo -u zulip /home/zulip/deployments/current/manage.py generate_realm_creation_link 
Error from server (BadRequest): container zulip is not valid for pod zulip-memcached-84ffbdf7d6-xr4nl
```

Reply:
Using `"app.kubernetes.io/name=zulip"` works: 

```
$ kubectl -n zulip exec -it $(kubectl get pods --namespace zulip -l "app.kubernetes.io/name=zulip" -o jsonpath="{.items[0].metadata.name}") -c zulip -- sudo -u zulip /home/zulip/deployments/current/manage.py generate_realm_creation_link
Please visit the following secure single-use link to register your 
new Zulip organization:

    https://zulip.varac-test.stackspin.net/new/...
```

Reply:
Thanks for the suggestion, fixed.